src/Data/Managers/V2/PostOrder/usePostOrder.ts

```diff
@@ -0,0 +1,45 @@
+import { UseMutationResult, useMutation } from "@tanstack/react-query";
+import { PrimaryBaseUrlManager, words } from "@/UI";
+import { InstanceForApi } from "@/UI/Components/Diagram/interfaces";
+import { useFetchHelpers } from "../helpers";
+
+/**
+ * React Query hook for sending an order batch from Instance Composer.
+ * @returns {Mutation} The mutation object for sending an order.
+ */
+export const usePostOrder = (
+  environment: string,
+): UseMutationResult<void, Error, InstanceForApi[], unknown> => {
+  const { createHeaders, handleErrors } = useFetchHelpers();
+  const headers = createHeaders(environment);
+  const baseUrlManager = new PrimaryBaseUrlManager(
+    globalThis.location.origin,
+    globalThis.location.pathname,
+  );
+  const baseUrl = baseUrlManager.getBaseUrl(process.env.API_BASEURL);
+
+  /**
+   * Sends an order with the provided bundle instances.
+   * @param {InstanceForApi[]} serviceOrderItems - The bundled instances to include in the order.
```

Comment:
Why not adjust the name of the typing too? To keep it consistent. Doesn't it match the typing used in the Order Overview page by any hasard?

Reply:
it is close, it's missing two properties, should I extend that interface?

Reply:
you can, yes, either you extend the interface and name it ComposerServiceOrderItems, or you add the missing properties as optional to the existing interface. Which properties are missing?

Reply:
related to the composer, so I will extend the interface